test/04-common.t

```diff
@@ -15,14 +15,13 @@ failure() {
     return 23
 }
 
-rc=0
-output=$(runcli success a b c 2>&1) || rc=$?
+try runcli success a b c
 is $rc 0 "runcli success"
-is "$output" "SUCCESS a b c" "runcli successful output"
+is "$got" "SUCCESS a b c" "runcli successful output"
 
-output=$(runcli failure a b c 2>&1) || rc=$?
+try runcli failure a b c
 is $rc 23 "runcli failure"
-like "$output", "test/04-common.t.*failure a b c.*oh noe" "runcli failure output"
+like "$got", "failure a b c.*oh noe" "runcli failure output"
```

Comment:
I noticed you removed the `test/04-common.t.*` from the match.
This was actually making sure that the error message contained information where it happened:
```
openqa-cli (34 test/04-common.t): 
```
With this PR it says:
```
openqa-cli (168 /home/tina/openqa-devel/repos/scripts/bpan/.bpan/lib/test-tap.bash):
```
So I assume it needs to go one caller level deeper somewhere. Not sure if that's possible with `try`.

Reply:
Right. There is stuff in test-tap-bash to adjust the call stack level on error messages.

I need to figure out a patch if you need this to have the file name.

BTW, just noticed the likely unintended comma in that line :)

I'll work on a fix for this.

Reply:
Fixed in da12258